src/messages/next_call_state_type_mismatch.cr

```diff
@@ -2,13 +2,15 @@ message NextCallStateTypeMismatch do
   title "Type Error"
 
   block do
-    text "The type of the state"
-    bold name
-    text "does not match of its definition in a"
-    bold "next call."
-  end
+    text "You were trying to assign an incompatible value to the status state."
+
+    text "\nThe type of the state is:"
```

Comment:
Use a `block` block instead of a new line since these messages are shown in the browser as well and it will not honour the new line since it's not displayed in a `pre`.

Reply:
Got it.